finance/management/commands/xformnetfilerawdata.py

```diff
@@ -273,22 +289,30 @@ def parse_ballot_info(row, locality, verbosity=1):
 def get_committee_benefactor(row):
     """Utility function to identify a committee benefactor.
     """
-    filer_id = clean_filer_id(row.get('cmte_Id'))
-    name = clean_name(row['tran_NamL'])
+    # $HACK; fields differ for contributions vs. expenditures!
+    if row.get('form_Type') == 'D':
+        filer_id = row.get('filerId') or row.get('filerLocalId') or row.get('filerStateId')
+        name = clean_name(row.get('filerName'))
+    else:
+        filer_id = clean_filer_id(row.get('cmte_Id'))
+        name = clean_name(row.get('cand_NamL') or row.get('tran_NamL') or row.get(''))
+
+    benefactor = None
 
     if filer_id is not None:
         # By ID
-        benefactor, _ = models.CommitteeBenefactor.objects.get_or_create(
+        benefactors = models.CommitteeBenefactor.objects.filter(
             filer_id=filer_id)
```

Comment:
Then maybe just using `.get` would work. Though this glosess over non-numeric/pending values.